Source/WebKit/UIProcess/playstation/WebProcessPoolPlayStation.cpp

```diff
@@ -32,11 +32,47 @@
 #include <wpe/wpe.h>
 #endif
 
+#if ENABLE(REMOTE_INSPECTOR)
+#include <JavaScriptCore/RemoteInspectorServer.h>
+#include <wtf/text/StringToIntegerConversion.h>
+#endif
+
 namespace WebKit {
 
+#if ENABLE(REMOTE_INSPECTOR)
+static std::optional<uint16_t> webInspectorServerPort()
+{
+    static std::optional<uint16_t> portNumber { std::nullopt };
+    static std::once_flag flag;
+    std::call_once(flag, [] {
+        char buffer[64];
+        if (!getenv_np("WebInspectorServerPort", buffer, sizeof(buffer)))
+            portNumber = parseInteger<uint16_t>(StringView::fromLatin1(buffer));
+    });
+
+    return portNumber;
+}
+
+static void initializeRemoteInspectorServer()
+{
+    if (Inspector::RemoteInspectorServer::singleton().isRunning())
+        return;
+
+    auto portNumber = webInspectorServerPort();
+    if (!portNumber)
+        return;
+
+    Inspector::RemoteInspectorServer::singleton().start(nullptr, portNumber.value());
```

Comment:
we don't want/need an `address`?  if we do want/need it, should we add it as a `WebInspectorAddress` envvar?

Reply:
Yes, we currently use only port and not address.

If we were to use both in the future, we would prefer to use the "ip:port" format and split them in the code, similar to wincairo.